### DISCUSSION TITLE
Installation Failed - Import Node

### DISCUSSION THREAD

**Discussion by Turki19** [Security-Onion-Solutions/securityonion#3654] [Unsupported Versions]
I have been trying to install the Import Node on my Mac, but with no success so far. 
I have Mac with 8GB and using VM Fusion to install SO. Everything goes well in the installation process till the last bit of the downloading operation; it freezes at (**Generating the CA and Checking In**) 
I tried to install both the Cent 7 OS and Ubuntu and ended with the same issue.

**Comment by weslambert** [comment]
You could try tailing `/root/sosetup.log` during install, but I would guess that maybe you still don't have enough CPU/RAM overall.  Have you tried seeing what you are using simply at idle using VMWare Fusion without running anything else? How many CPU cores are available for the VM?
